packages/domains-table/src/domains-table/domains-table-status-cell.tsx

```diff
@@ -0,0 +1,37 @@
+import { ResponseDomain } from '@automattic/data-stores/src/domains/types';
+import { useTranslate } from 'i18n-calypso';
```

Comment:
We're using `__()` from `@wordpress/react-i18n` in this package. Hopefully `resolveDomainStatus` doesn't need to be adapted too much to support that 🤞

Reply:
This doesn't seem a simple change, we will have to revisit it